src/main/java/net/sf/mzmine/modules/masslistmethods/listexport/help/help.html

```diff
@@ -0,0 +1,98 @@
+<html>
```

Comment:
Apparently it was.  It is in the Beta code directory.

On Fri, Oct 9, 2015 at 9:47 PM, TDI-tenderholt <notifications@github.com>
wrote:

> In
> src/main/java/net/sf/mzmine/modules/masslistmethods/listexport/help/help.html
> <https://github.com/TDI-tenderholt/mzmine2/pull/1#discussion_r41664838>:
>
> > @@ -0,0 +1,98 @@
> > +<html>
>
> Was this added in the MZminePI code? It doesn't look like anything Dan
> would have written, but it seems to be help for the listexport module
> that he wrote.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/TDI-tenderholt/mzmine2/pull/1/files#r41664838>.
>